cfme/networks/views.py

```diff
@@ -545,7 +545,7 @@ class SubnetAddView(BaseLoggedInPage):
     title = Text('//div[@id="main-content"]//h1')
     network_manager = Select(id='ems_id')
     cloud_tenant = Select(name='cloud_tenant_id')
-    network = Select(name='cloud_network_id')
+    network = Select(name='network_id')
```

Comment:
**QUESTION**
Does it work for 5.8 as well?

Reply:
yes, it was tested on 5.8

Reply:
Works for 5.8 and 5.9. locator name was changed on both